### DISCUSSION TITLE
视图能力是否可以尝试加入“不包含”

### DISCUSSION THREAD

**Discussion by QutantumProgrammer** [Tencent/bk-ci#5079] [Ideas]
![image](https://user-images.githubusercontent.com/19562478/131851598-c4ab8fa5-1469-4ef5-a0b8-3f092ab2d5e2.png)
流水线多的时候使用不包含进行判断会更容易做分类

**Comment by iccengan** [comment]
通过给流水线打标签的方式，用流水线标签归类能否满足需求？这样的使用方式也更规范，也能覆盖绝大部分的场景。
